### DISCUSSION TITLE
Don't get freerouting KiCad Plugin started

### DISCUSSION THREAD

**Discussion by kjack63** [freerouting/freerouting#199] [General]
![freerouting_error](https://github.com/freerouting/freerouting/assets/109347625/98aa58cf-b8e7-4ad7-854f-9befe3a65369)

Hello,
any idea why I got this error when starting the freerouting plugin in KiCad7

Using KiCad 7.0.2; Freerouting 1.8.0 and Temurin JRE 17

**Comment by andrasfuchs** [comment]
According to ChatGPT:
```
The exit code 3221225786 indicates that your Java application encountered a system exception, which caused it to terminate unexpectedly. This exit code is specific to Windows systems and corresponds to the hexadecimal value 0xC000013A.

This error code usually indicates an application crash due to a stack overflow or a heap corruption.
```

Could you upload the KiCad project or the DSN file?

**Reply by kjack63** [reply]
Hello, I've added in the zip two files, one manual exported DSN and the temp file created during using the freerouting plugin.
One more information which might help. If I'm using a portable version of Freerouting, I have trouble to load the DSN file.
[temp-freerouting.zip](https://github.com/freerouting/freerouting/files/11566502/temp-freerouting.zip)

**Comment by andrasfuchs** [comment]
I could load your file without the stack overflow error on Windows 11 with Freerouting 1.8 (using Temurin JDK 17.0.5).

Unrelated to the stack overflow you have, KiCad 7.0.2 has a bug regarding the DSN export as you can see:
![image](https://github.com/freerouting/freerouting/assets/910321/27d69b79-0c57-4b38-a892-d1653dde9056)

I'm not too confident it will solve the issue, but could you update KiCad to 7.0.4 and try again?

**Reply by kjack63** [reply]
It looks like, that 7.0.4 isn't available for Download currently. 
See discussion here: https://forum.kicad.info/t/stable-release-7-0-4/42615

**Reply by kjack63** [reply]
Another point, if I use the standalone version of Freerouting (1.8.0), then the process always ends with Postroute finished (not sure if correct, using German language). Any idea why?

Edit: I tested the installed standalone version with a KiCad6 DSN file, this was working. It seems, that Freerouting has a problem with KiCad 7 versions.

**Comment by andrasfuchs** [comment]
Interesting, I could download 7.0.4 a few days ago, but now it's gone from https://downloads.kicad.org/kicad/windows/explore/stable.
As I can see on the discussion link I'm not the only one confused :)

The "Postroute finished" is normal, but you have a good point, it should be something more meaningful indicating successful completion.

**Reply by kjack63** [reply]
I'm afraid, I need to wait for a new KiCad7 release. Until then, I guess I'll have to exercise patience, unless I want to reload the old KiCad version ;-) .

**Reply by kjack63** [reply]
Hello,  good news. I've updated to KiCad 7.0.5, to be sure I reinstalled the Freerouting Plugin and it works now.

**Reply by andrasfuchs** [reply]
Great, thank you for the confirmation!
